src/MusicStore/Areas/Admin/Controllers/StoreManagerController.cs

```diff
@@ -130,19 +146,19 @@ public IActionResult Create()
         // POST: /StoreManager/Edit/5
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Edit(Album album)
+        public async Task<IActionResult> Edit(CancellationToken requestAborted, Album album)
```

Comment:
Sure. I'll make it as the last parameter in the case actions doesn't use the default parameter values..